lib/modules/apostrophe-login/index.js

```diff
@@ -349,12 +353,24 @@ module.exports = {
       return next();
     };
 
+    // Check if the first character about the option `loginUrl` is `/`
+    // and if is not present, add the `/` in the first position.
+    // If the option is not present, return the default url `/login`
+    self.fetchLoginUrl = function() {
+      if (self.options.loginUrl) {
+        return self.options.loginUrl && self.options.loginUrl.startsWith('/') ? self.options.loginUrl : `/${self.options.loginUrl}`
```

Comment:
We should skip this mechanism, the developer is responsible for including the leading / on all other URL based options and I don't want to cause any confusion about that.